### ISSUE TITLE
Determine how to track AWS resources we use which aren't created automatically

### ISSUE THREAD

**Issue by justincc** [ebi-ait/hca-ebi-dev-team#22]
These are resources not created by terraform, etc.

One possibility is that we get AWS to tell us via what is tagged. Another is to track this manually.

**Acceptance criteria**
* [x] The process or the resources are documented [here](https://docs.google.com/document/d/1R2ZrMNbZ5TqJz7_DGiZgNPN2KF-dfsN9ETNq7ofX5_E/edit#heading=h.jieeeeca1b0w)

**Comment by rolando-ebi** [comment]
Kubernetes also creates AWS resources (EBS volumes, load-balancers). 

I believe there are annotations we can put on our k8s service resource .yamls to apply a tag to the underlying AWS load-balancer(https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#load-balancers)

K8s does not have a way to apply tags to provisioned volumes though, looks like. See https://github.com/kubernetes/kubernetes/issues/50898

One possible option is to have terraform pre-create and tag AWS resources that kubernetes tries to create, then supply references to those pre-created AWS resources when creating kubernetes services/persistent-volumes. I believe there isn't much support for this either though(https://github.com/kubernetes/kubernetes/issues/10323)

**Comment by rolando-ebi** [comment]
@prabh-t pointed out that our AWS resources created by k8s get auto-tagged as specified in the terraform auto-scaling group. Looks like k8s somehow knows to add them to the auto-scaling group, or perhaps this is functionality provided by EKS. The above comment can be ignored

**Comment by prabh-t** [comment]
I can confirm in the case of autoscaling group, tags are propagated at launch (this is specified in the definition) to any "children" resources created by Terraform for ASG. I've verified this, and found EC2 instances and EBS volumes attached to them have the appropriate tags attached. I've also added the default tags to this resource type.
A list of resources not created automatically is now documented.
I looked into automatically updating the dns records when a cluster is rebuilt, but could not get obtain the dns name/IP of the ALB spawned by ASG in terraform. We may be able to get this by explicitly defining the load balancer and link it to the ASG. It should be possible to contain aws_lb.ingest_eks.dns_name and use it to set aws_route53_record.
